### DISCUSSION TITLE
Offer diskusage functionality in File-Manager

### DISCUSSION THREAD

**Discussion by PanderMusubi** [FossifyOrg/General-Discussion#230] [Ideas]
Functionality offered by https://github.com/IvanVolosyuk/diskusage is very useful for graphically finding files and directories that take up a lot of disk space and can be removed. See https://github.com/IvanVolosyuk/diskusage and https://f-droid.org/en/packages/com.google.android.diskusage/

Personally, I use it a lot. However, the maintenance of this app is minimal. Perhaps it is better to have File-Manager offer something similar. Especially deleting (trash can) and moving files could be handled by existing functionality.

**Comment by Aga-C** [comment]
Can you raise it as a [feature request in File Manager](https://github.com/FossifyOrg/File-Manager/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml)? Thanks!
